platform/viewer/src/index.js

```diff
@@ -24,6 +24,7 @@ import ReactDOM from 'react-dom';
  */
 import OHIFVTKExtension from '@ohif/extension-vtk';
 import OHIFDicomHtmlExtension from '@ohif/extension-dicom-html';
+import OHIFDicomSegExtension from '@ohif/extension-dicom-seg';
```

Comment:
Wondering if we need to split so that we have "Segmentation Loading" as one extension, and "Segmentation Tools" as a separate extension.

We've also discussed use `sopClassHandler` for "filtering the thumbnails" and a new hook, when the visible viewports show new displaySets/studyInstanceUids, that extensions should be given a hook to load new data.

Can we offload this to `@ohif/core` in some way? Is this part of the slicer `IO Module` pattern we've seen?

Reply:
Splitting them up may not be a bad idea. Someone may want to be able to view and not edit.

> that extensions should be given a hook to load new data.

Yeah it still feels a bit wonky to do it this way, but that seemed like the easiest way to keep the loading encapsulated within the plugin to some degree.

We already have `loadAndCachedDerivedDisplaySets` in `@ohif/core`, what else do you think should be pushed down there? We don't have much more in the seg extension other than the loader itself.